### DISCUSSION TITLE
Direct connection with RS485  sensors

### DISCUSSION THREAD

**Discussion by garudaonekh** [eModbus/eModbus#262] [General]
Hi, 
I normally connect RS485 sensors to my ESP32 through below module. Is it possible to connect directly without below module using this library?

![-266567182510340880](https://user-images.githubusercontent.com/2074857/215243476-26fa1bcf-e80b-4c9a-97b0-9e02d37a1e6d.jpg)

**Comment by Miq1** [comment]
That depends on the sensor - can it do serial comms without RS485? - and the situation where you are using it. Basically RTU is a regular serial communication on the logic side, but RS485 is a different transmission technique underneath. It is chosen for a better protection against external influences like electrical or magnetic fields and is using a voltage difference logic instead of the DC HIGH/LOW signals of a serial communication.

**Reply by garudaonekh** [reply]
No, it cannot. It is RS485. It is 5v
